docs/tutorial/t-manage-passwords.md

```diff
@@ -121,6 +137,7 @@ juju run data-integrator/leader get-credentials
 ```
 
 Running the command should now output a different password:
+
```

Comment:
**question:** What's the reason for these gaps? Doesn't markdown rendering handle that already?

Reply:
It is a best practice to envelop all blocks (code blocks, lists, tables, paragraphs of texts, headings) in empty lines. That helps render markdown correctly in most implementations. Not following this simple rule can create some unpredictable behaviour in some implementations of markdown parsers.